src/WebServer/Http/HttpServer.cs

```diff
@@ -19,9 +20,11 @@ public class HttpServer : IDisposable
         private readonly StreamSocketListener _listener;
         private readonly SortedSet<RouteRegistration> _routes;
         private readonly ContentEncoderFactory _contentEncoderFactory;
+        private ILogger _log;
 
         public HttpServer(int serverPort)
         {
+            _log = LogManager.GetLogger<HttpServer>();

```

Comment:
Yes, but we could add IsErrorEnabled(), nothing to do with log levels but just if a certain log function logs it out or not, I think it could be useful, do you have a lot of problems with me adding it?